real_check/real_check.ps1

```diff
@@ -70,7 +70,14 @@ function Check-Symbols {
         return $return_code
 }
 
-exit (Check-Symbols -lib $args[0])
+ForEach($lib in $args){
+    Write-Host "Checking lib $lib"
+    $exit_code = Check-Symbols -lib $lib
```

Comment:
>-lib [](http://example.com/codeflow?start=31&length=4)

<not a powershell wizard>, but Check-Symbols doesn't seem to have a named parameter called "lib". </not a powershell wizard>

Reply:
This blog seems to suggest that passing the -lib flag is optional when a function parameter is declared using params(). 

[https://devblogs.microsoft.com/scripting/named-arguments-for-powershell-functions-best-practices/](https://devblogs.microsoft.com/scripting/named-arguments-for-powershell-functions-best-practices/)

Might be best to keep the flag for clarity.